AnnoyancesFilter/Popups/sections/popups_allowlist.txt

```diff
@@ -44,6 +44,8 @@ g123.jp,accounts.binance.com#@##credential_picker_container
 !
 @@||my.leadpages.net/template/$script,domain=weddingjams.com|thesaturdaypaper.com.au|careercontessa.com
 @@||my.leadpages.net^*/public/
+! https://github.com/AdguardTeam/AdguardFilters/issues/177450
+@@||shop.blu-blumen.de^$script,xmlhttprequest
```

Comment:
I think it's too broad and it would be better to add a more specific rule, for example:
```adblock
@@||shop.blu-blumen.de/Scripts/CookieBanner.js
```
Please also move it to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

By the way, it unblocks cookie notice, so we should fix it too.
It looks like that it could also be fixed by a scriptlet:
```adblock
shop.blu-blumen.de#%#//scriptlet('set-constant', 'cb_isMarketingAllowed', 'noopFunc')
```

So we could add exception for platforms which do not support scriptlets, like:
```adblock
!+ PLATFORM(ios, ext_android_cb, ext_safari)
@@||shop.blu-blumen.de/Scripts/CookieBanner.js
```
to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt
<hr>

And:
```adblock
shop.blu-blumen.de#%#//scriptlet('set-constant', 'cb_isMarketingAllowed', 'noopFunc')
```
to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt

Reply:
Thank you for the detailed comment, made changes